lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -225,6 +230,7 @@ def register_task(self, family, task_role_arn, network_mode, container_definitio
         params = dict(
             family=family,
             taskRoleArn=task_role_arn,
+            executionRoleArn=execution_role_arn,
```

Comment:
executionRoleArn must not be set if it's not required, otherwise anyone relying on this module will need to upgrade to a very recent botocore.